demo/server/src/modules/keycloak/index.ts

```diff
@@ -1,26 +1,38 @@
+import { EndpointSecurity } from '@raincatcher/express-auth';
 import * as express from 'express';
 import * as session from 'express-session';
 
 import appConfig from '../../util/Config';
-import sessionOpts from '../SessionOptions';
 
 // tslint:disable-next-line:no-var-requires
 const Keycloak = require('keycloak-connect');
+import sessionOptions from '../SessionOptions';
 
 const config = appConfig.getConfig();
 
-export function init(app: express.Express) {
-
-  // Express Session Configuration.
-  app.use(session(sessionOpts));
-
-  // Create a session store
-  const memoryStore = new session.MemoryStore();
-  const keycloak = new Keycloak({ store: memoryStore }, config.keycloakConfig);
-
-  // Use keycloak middleware & define applications logout route
-  app.use(keycloak.middleware({logout: '/logout'}));
+export class KeycloakSecurity implements EndpointSecurity {
+  // Add typescript definitions
+  private keycloak: any;
+
+  public init(app: express.Express, sessionOpts: session.SessionOptions): void {
+    // Express Session Configuration.
+    app.use(session(sessionOpts));
+
+    // Create a session store
+    const memoryStore = new session.MemoryStore();
+    this.keycloak = new Keycloak({ store: memoryStore }, config.keycloakConfig);
+  }
```

Comment:
also re-add the `app.use(keycloak.middleware({logout: '/logout'}));` to use keycloak middleware and define the logout route